issue_template.md

```diff
@@ -0,0 +1,15 @@
+## Expected Result
+
+
+
+
+## Actual Result Gotten
+
+
+
+## Ways to Resolve this issue
+
+
+
+
+## Information about Device Issue was experience
```

Comment:
## Additional Information
- [ ] MessageKit Version
- [ ] iOS Version(s)
- [ ] Swift Version
- [ ] Devices/Simulators
- [ ] Reproducible in ChatExample? (Yes/No)